src/fight/mod.rs

```diff
@@ -39,15 +39,15 @@ fn compute_mean_case(
     attacking_regiment: &regiment::Regiment,
     defending_regiment: &regiment::Regiment,
 ) -> (usize, f64) {
-    let attacking_stats: &model::Stats = attacking_regiment.get_model().get_stats();
-    let defending_stats: &model::Stats = defending_regiment.get_model().get_stats();
+    let attacking_stats: &model::Stats = &attacking_regiment.get_model().get_stats();
+    let defending_stats: &model::Stats = &defending_regiment.get_model().get_stats();
```

Comment:
Why this change ?

Reply:
In the function I was creating a copy of the stats struct but finally changed my way of handling the modifier so this change is no longer existing